packages/components/package.json

```diff
@@ -95,7 +100,7 @@
       "react-app/jest"
     ],
     "rules": {
-      "no-console": "error"
+      "no-console": "warn"
```

Comment:
Why do we need this change? you don't seem to add anymore console.logs. Are they included in generated code?

Reply:
Oh, no, that creeped into staging D:

I still do a lot of testing locally, and did use that locally